### PR TITLE
neofetch 1.4 (new formula)

### PR DESCRIPTION
**Pull request by iandrewt**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

added neofetch script, a fast, highly customisable system info script from @dylanaraps

### CODE REVIEW

File: Library/Formula/neofetch.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/dylanaraps/neofetch"
+  url "https://github.com/dylanaraps/neofetch/archive/1.4.tar.gz"
+  sha256 "b0bda1ac5c3fa4357c6c2157767bcbf08d0e231e3d5e15c4408490f1c3175756"
+  head "https://github.com/dylanaraps/neofetch.git"
+
+  def install
+    bin.install "neofetch"
+    prefix.install "ascii"
+    prefix.install "config"
+    man.install "neofetch.1"
```
Comment (bfontaine) [inline Library/Formula/neofetch.rb]: Can’t we use the Makefile here?

**Comment by iandrewt** [comment]
This is why I shouldn't write formulas from my phone at midnight... Fixed, and that fixed the ascii distro logo too it seems.

File: Library/Formula/neofetch.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/dylanaraps/neofetch/archive/1.4.tar.gz"
+  sha256 "b0bda1ac5c3fa4357c6c2157767bcbf08d0e231e3d5e15c4408490f1c3175756"
+  head "https://github.com/dylanaraps/neofetch.git"
+
+  def install
+    system "PREFIX=#{prefix} make install"
+  end
+
+  test do
+    system "neofetch"
```
Comment (bfontaine) [inline Library/Formula/neofetch.rb]: Please use `"#{bin}/neofetch"` instead of `"neofetch"`.

File: Library/Formula/neofetch.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class Neofetch < Formula
+  desc "fast, highly customisable system info script"
+  homepage "https://github.com/dylanaraps/neofetch"
+  url "https://github.com/dylanaraps/neofetch/archive/1.4.tar.gz"
+  sha256 "b0bda1ac5c3fa4357c6c2157767bcbf08d0e231e3d5e15c4408490f1c3175756"
+  head "https://github.com/dylanaraps/neofetch.git"
+
+  def install
+    system "PREFIX=#{prefix} make install"
```
Comment (bfontaine) [inline Library/Formula/neofetch.rb]: Please separate each argument: `system "make", "install", "PREFIX=#{prefix}"`.

**Comment by DomT4** [comment]
It's like `archey` in functionality, interesting!

Merged in https://github.com/Homebrew/homebrew/commit/eec9e62ae950058f8d0ab127aa98b11c2acea1ab. Thank you for another contribution to Homebrew @iandrewt! :smiley_cat:

**Pull request closed without merging**